devdoc/EndUsers.md

```diff
@@ -2,14 +2,115 @@
 
 Welcome to the End User Documentation. This guide provides a comprehensive collection of resources and instructions created by developers to assist end-users in effectively utilizing the platform.
 
+## General HPCC Platform Documents
+
+[Version Support](VersionSupport.md)
+Details on Supported Versions
+
+[Platform Overview](../README.md) Overview of the HPCC Systems Platform.
+
+[File Processing](NewFileProcessing.md)
+Guide for implementing new file processing features.
+
+[Memory Manager](MemoryManager.md) Documentation on memory management strategies.
+
+[Data Masking](../system/masking/include/readme.md)
+Documentation for the data obfuscation framework.
+
+[DataMasker plugin](../system/masking/plugins/datamasker/readme.md)
+Documentation for DataMasker plugin.
+
+[ESP API tool](../tools/esp-api/README.md)
+Documentation for ESP API tool.
+
+## Azure
+
+Details about using the HPCC Systems platform in Azure.
+
+[Azure Tips And Tricks](./userdoc/azure/TipsAndTricks.md)
+General Azure deployment tips for HPCC Systems Platform.
+
+## ECL Bundles
+
```

Comment:
remove blank line for consistency

Reply:
Headings should be surrounded by blank lines.